README.md

```diff
@@ -28,6 +35,9 @@ npm run deploy_local
 
 # npx hardhat run scripts/deploy.js --network goerli
 npm run deploy_goerli
+
+# npx hardhat verify --network goerli {YOUR_IMPLEMENTATION_ADDRESS}
+npm run verify_goerli {YOUR_IMPLEMENTATION_ADDRESS}
```

Comment:
This command is **AMAZING**! It takes care of verifying the contracts bytecode in Etherscan for you, so easy!